### PR TITLE
fix img issue

### PR DESCRIPTION
**Pull request by spcognizant**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -557,6 +557,277 @@ a:hover {
...
+  overflow: hidden;
+  /* margin: auto; */
+  width: 100%;
+  -webkit-transition: height 0.15s ease-in;
+  -moz-transition: height 0.15s ease-in;
+  -ms-transition: height 0.15s ease-in;
+  -o-transition: height 0.15s ease-in;
+  transition: height 0.15s ease-in;
+}
+.carousel .slider-wrapper.axis-horizontal .slider {
```
Comment (sophia-cg) [inline src/App.css:702]: surely you can collapse some of these ids? There's a lot of duplication.

E.g. make it 
.carousel .slider-wrapper.axis-vertical, .carousel .slider-wrapper.axis-horizontal .slider {
   xyz
}

**Review by sophia-cg — APPROVED** [review]
It is goodo

**Review by Bentley-Turner — APPROVED** [review]
Nice Stuff buddy!

File: src/App.css (hunk 1 above)
Comment (spcognizant) [reply src/App.css:702]: I copied from that library :P

**Pull request merged**